TodoListService/App_Start/Startup.Auth.cs

```diff
@@ -30,6 +30,7 @@ public void ConfigureAuth(IAppBuilder app)
             app.UseActiveDirectoryFederationServicesBearerAuthentication(
                 new ActiveDirectoryFederationServicesBearerAuthenticationOptions
                 {
+                    // [jelled] Why is Audience not being checked via this mechanism?
```

Comment:
[tvuylsteke] 4 lines above the validAudience is populated with the configuration file version. I should refactor this. Pointless usage of variables.